Gemfile.lock

```diff
@@ -124,4 +124,4 @@ RUBY VERSION
    ruby 2.6.2p47
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```

Comment:
This is unrelated to your PR, furthermore, upgrading this will cause the builds to fail.
Please revert/remove this change.